tools/update-tester/src/testrunner.rs

```diff
@@ -362,3 +454,240 @@ fn get_values(query_results: Vec<SimpleQueryMessage>) -> QueryValues {
         })
         .collect()
 }
+
+// Functions below this line are originally from sql-doctester/src/runner.rs
```

Comment:
We should consider isolating duplicated functionality to separate files.  Is that too painful?

Reply:
I've updated the release automation plan to include next steps for what could be done to yank out all of the duplicated functionality. Even though the functions are the same, they are using slightly differing versions of Test and TestError. We would need to update parts of sql-doctester and update-tester in order to get them working nicely together with this code which shouldn't be hard but might be worth doing in its own PR.